effektif-workflow-api/src/main/java/com/effektif/workflow/api/condition/Not.java

```diff
@@ -40,17 +40,16 @@ public boolean isEmpty() {
 
   @Override
   public void readBpmn(BpmnReader r) {
-    for (XmlElement andElement : r.readElementsEffektif(getClass())) {
-      r.startElement(andElement);
-      condition = r.readCondition();
-      r.endElement();
-    }
+    condition = r.readCondition();
+    r.endElement();
   }
 
   @Override
   public void writeBpmn(BpmnWriter w) {
     w.startElementEffektif(getClass());
-    condition.writeBpmn(w);
+    if (condition != null) {
+      condition.writeBpmn(w);
+    }
```

Comment:
The condition already implements `isEmpty` which represents that kind of information. We could skip empty the creation of empty elements.